docs/source/basicusage.rst

```diff
@@ -17,7 +17,7 @@ Show help with all available arguments:
 
 ::
 
-    $ python prompt.py -h
+    $ python np-prompt -h
```

Comment:
I don't think we want `python` here, do we?

Reply:
0380ad6

Reply:
https://github.com/CityOfZion/neo-python/commit/bf8f00c0d9f559b9459b76422d0b2f9ce6969f2d 👍 😁